cmake/OpenCVFindTimVX.cmake

```diff
@@ -0,0 +1,83 @@
+set(OPENCV_LIBTIMVX_ROOT_DIR "" CACHE PATH "Path to libtim-vx installation")
+set(TIMVX_SDK_DIR "" CACHE PATH "Path to Tim-VX SDK")
+set(TIMVX_SDK_LIB_CANDIDATES "ArchModelSw;CLC;Emulator;GAL;NNArchPerf;OpenVXC;OpenVX;OpenVXU;vdtproxy;VSC" CACHE STRING "Tim-VX's SDK library candidates list")
+
+function(find_timvx_sdk_libs _found)
+    foreach(one ${TIMVX_SDK_LIB_CANDIDATES})
+        find_library(TIMVX_${one}_LIBRARY ${one} PATHS "${TIMVX_SDK_DIR}/lib")
+        if(TIMVX_${one}_LIBRARY)
+            list(APPEND _sdk_libs ${TIMVX_${one}_LIBRARY})
+        endif()
+    endforeach()
+    list(APPEND _sdk_libs ${TIMVX_SDK_DIR}/lib/libOpenVX.so.1)
+    set(${_found} ${_sdk_libs} PARENT_SCOPE)
+endfunction()
+
+if(OPENCV_LIBTIMVX_ROOT_DIR  AND NOT BUILD_TIMVX)
+    message(STATUS "Tim-VX: Use binaries at ${OPENCV_LIBTIMVX_ROOT_DIR}")
+
+    set(TIMVX_FOUND ON)
+    set(BUILD_TIMVX OFF)
+
+    set(TimVX_INCLUDE_DIR "${OPENCV_LIBTIMVX_ROOT_DIR}/include" CACHE PATH "Tim-VX include directory")
+    set(TimVX_LIB "${OPENCV_LIBTIMVX_ROOT_DIR}/lib/libtim-vx.a" CACHE PATH "Path to Tim-VX library")
+    # set(TimVX_LIB "${OPENCV_LIBTIMVX_ROOT_DIR}/src/tim/vx/internal/libtim_internal.a" CACHE PATH "Path to Tim-VX library")
+    list(APPEND TimVX_LIB "${OPENCV_LIBTIMVX_ROOT_DIR}/../src/tim/vx/internal/libtim_internal.a")
```

Comment:
This line is added to link `libtim_internal.a` as well. Currently there is a linking bug in TIM-VX where `libtim_internal.a` is not correctly linked into `libtim-vx.a`. We have informed guys from Verisilicon to fix this.